### ISSUE TITLE
cookie error after upgrade

### ISSUE THREAD

**Issue by jseutens** [Alanaktion/phproject#345]
500 Internal Server Error
session_set_cookie_params() expects at most 5 parameters, 6 given

/tmp/cache is empty 
apc cache is cleared (server and fpm where also restarted)

error log entry : AH01071: Got error 'PHP message: session_set_cookie_params() expects at most 5 parameters, 6 given\nPHP message: [app/model/config.php:32] Base->set()\nPHP message: [index.php:83] Model\\Config::loadAll()\n'

**Comment by jseutens** [comment]
I did a complete re-installation and added back my users and projects from a DB backup.
All working now, this is not a code error but a value that is in the config table that doens't belong there anymore (ASSUMPTION!) (with a almost empty config table i could login to the broken upgrade site)

**Comment by Alanaktion** [comment]
This is fixed in version 1.7.3.

**Comment by Herbb** [comment]
Today I was updating phproject from version 1.6.2 up to the latest 1.7.3.
After upgrade I had the same problem, cleared cache but this didn't help, purged 'session' table which did not help as well.
I found in db/database.sql how table should be created in current version, so I recreated it and was surprised it did not help as well, but I noticed

``` INSERT INTO `config` (`attribute`, `value`) VALUES ('version', '18.02.07');```

at the end of the file. I checked value in my database just to notice it's value is "15.04.06".
I decided to change it to current version for no real reason, but after this change my issue with sessions is gone.
 ``` UPDATE `config` SET version='18.02.19' WHERE attribute='version'; ```
